avocado/utils/software_manager.py

```diff
@@ -26,13 +26,6 @@
 and multiple high level package managers (here called backends). If you want
 to make this lib to support your particular package manager/distro, please
 implement the given backend class.
-
-:author: Higor Vieira Alves <halves@br.ibm.com>
-:author: Lucas Meneghel Rodrigues <lmr@redhat.com>
-:author: Ramon de Carvalho Valle <rcvalle@br.ibm.com>
-
-:copyright: IBM 2008-2009
```

Comment:
We're missing this line on the comment section though, needs to be added there.

Reply:
Will fix on v2.